Led.cpp

```diff
@@ -1,33 +1,51 @@
 #include "Led.h"
 
-Led::Led() : pin(pin) {
+Led::Led() : Pin(Pin) {
 }
 
-Led::Led(unsigned pin) : pin(pin) {
+Led::Led(unsigned pin) : Pin(pin) {
     //	pinMode(pin, OUTPUT);
 }
 
 void Led::setBrightness(unsigned percent) {
-    //	analogWrite(led, precent);
+    //  percent = map(percent, 0, 100, 0, 255);
```

Comment:
This is correct but can you think of a way without relying on the built in function? Hint, since we are going from a range [0,A] to a range [0,B] this is a special case that resolves to just a constant multiplier.